### PR TITLE
Improve fireworks theme

### PR DESCRIPTION
**Pull request by atjn**
Rewrite the fireworks theme to improve a number of things:

- There is no longer a flash of unstyled content when loading the page.
- If you don't have JS enabled, you still get a dark stregsystem.
- Lots of optimizations to the rendering of the rockets.
- Minor optimizations to the JS code.

I would like to do a bit more work on this in the future, but I am currently too busy and these fixes are already very good on their own.

### CODE REVIEW

**Comment by mads256h** [comment]
bro addede tabs :skull:

**Comment by atjn** [comment]
Jeg kan godt lade være med at ændre formatting, jeg gør det bare fordi frontenden bruger 20 forskellige formateringsstandarder, så jeg har forsøgt altid at bruge den samme formatering i det kode jeg ændrer.

Github understøtter at du kan ignorere whitespace-ændringer. Er det godt nok til at du kan læse det?:
https://github.com/f-klubben/stregsystemet/pull/538/files?w=1

File: stregsystem/static/stregsystem/themes/fireworks/fireworks.js [hunk 1]
```diff
@@ -26,319 +20,349 @@ const minSpaceFromSides = 150;
...
 }
 
 function runAnimations(now) {
-  updateRocket(now);
-  updateExplosions(now);
+	updateRocket(now);
+	updateExplosions(now);
 
-  window.requestAnimationFrame(runAnimations);
+	requestAnimationFrame(runAnimations);
```
Comment (mads256h) [inline stregsystem/static/stregsystem/themes/fireworks/fireworks.js:214]: Non-blocking NITPICK: I like using the window prefix so it is clear this function is not defined in this code

**Review by mads256h — APPROVED** [review]
LGTM

**Pull request merged**